reporter/src/main/kotlin/reporters/NoticeByPackageReporter.kt

```diff
@@ -20,6 +20,7 @@
 package org.ossreviewtoolkit.reporter.reporters
 
 import java.io.File
+import java.util.SortedMap
```

Comment:
I'm surprised this commit does not require any changes to expected test output. Did you verify that this commit alone indeed passes the tests?

Reply:
Yes it does. I assume licenses are not always unsorted, only if some aggregation happens, but I didn't investigate this.